up.conf partially ignored and wrong location

Latest version 2.5.0 running on Ubuntu 20.04. Remote Unifi Controller

unpoller is still looking for up.conf in /etc/unifi-poller/up.conf and will not start unless the file is there

unifi.default and unifi.controller settings are ignored and appear to be hardcoded to https://127.0.0.1:8443

Log:

`Dec 22 09:38:51 dash unpoller[174578]: 2022/12/22 09:38:51 [ERROR] finding file: stat /etc/unifi-poller/up.conf: no su│
ch file or directory
Dec 22 09:38:51 dash systemd[1]: unpoller.service: Main process exited, code=exited, status=1/FAILURE                 │zc -u jim@koti.tech -p w3ZsreTuPL5C#ke
Dec 22 09:38:51 dash systemd[1]: unpoller.service: Failed with result 'exit-code'. `


I have installed unpoller locally on a separate Unifi controller and so long as I place the conf file in /etc/unifi-poller/up.conf it all works so it looks like /etc/unifi-poller/up.conf and https://localhost:8443 are hardcoded somewhere.

Jim

--- a/init/systemd/unpoller.service
+++ b/init/systemd/unpoller.service
@@ -9,7 +9,7 @@ After=network.target
 Requires=network.target
 
 [Service]
-ExecStart=/usr/bin/unpoller $DAEMON_OPTS
+ExecStart=/usr/bin/unpoller -c /etc/unpoller/up.conf $DAEMON_OPTS
 EnvironmentFile=-/etc/default/unpoller
 EnvironmentFile=-/etc/sysconfig/unpoller
 Restart=always

